### ISSUE TITLE
Metric_Instances.ModelVersion is null when pulled via API

### ISSUE THREAD

**Issue by volkcj** [NorthArrowResearch/champ-workbench#44]
There is no model version information associated with metrics when pulled via API.  Not sure if this is a bug or an enhancement?

**Comment by philipbaileynar** [comment]
We tried discussing this as a group several times and no consensus could be reached about what "model version" actually meant. 

Given the importance, we have just made implemented a solution to get something working (given that this field is required for the validation report to work):

* All engines that produce metrics that go into a schema now include a `ModelVersion` number in the meta data at the top of the result XML file.
* This version number is a manually set by the model owner. It is their responsibility to change this version when they fix bugs or implement changes to metric calculation methodologies.
* There is one `ModelVersion` value for an entire model/engine. We are NOT tracking version numbers down to the individual metrics. i.e. Topo metrics has a `ModelVersion` number but "Site Length does not.
* The `ModelVersion` value is scraped along with metrics and inserted into the API as a field in all metric schemas. This is the same approach used for the `GenerationDate` on the metric schemas.
* Note that this `ModelVersion` is not defined in the metric schema in the Workbench. i.e. there doesn't need to be a metric defined for ModelVersion.
* The Workbench is already expecting this `ModelVersion` and will consume it once it is available on metric schemas.
* This solution will go live during the next full model engine run.
